sympy/physics/mechanics/functions.py

```diff
@@ -399,20 +400,44 @@ def Lagrangian(frame, *body):
     return kinetic_energy(frame, *body) - potential_energy(*body)
 
 
-def find_dynamicsymbols(expression, exclude=None):
+def find_dynamicsymbols(expression, exclude=None, reference_frame=None):
     """Find all dynamicsymbols in expression.
 
+    If the optional ``exclude`` kwarg is used, only dynamicsymbols
+    not in the iterable ``exclude`` are returned.
+    If we intend to apply this function on a vector, the optional
+    ''reference_frame'' is also used to inform about the corresponding frame
+    with respect to which the dynamic symbols of the given vector is to be
+    determined.
+
+    Parameters
+    ==========
+
+    expression : sympy expression
+
+    reference_frame : ReferenceFrame
+        The frame with respect to which the dynamic symbols of the
+        given vector is to be determined.
+
+    exclude : iterable of dynamicsymbols
```

Comment:
Can you please document them in the order in which they appear in the API? i.e. first `expression`, then `exclude` and lastly `reference_frame`.

Also use this format for `exclude` and `reference_frame`: 
```
exclude : iterable of dynamicsymbols, optional

reference_frame : ReferenceFrame, optional
    The frame with respect to which the dynamic symbols of the
    given vector is to be determined.
```